tests/Bedrock.Framework.Benchmarks/HttpHeaderReaderBenchmarks.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Bedrock.Framework.Protocols.Http.Http1;
+using BenchmarkDotNet.Attributes;
+using System;
+using System.Buffers;
+using System.Text;
+
+namespace Bedrock.Framework.Benchmarks
+{
+    public class HttpHeaderReaderBenchmarks
+    {
+        [Params(10, 1000)]
+        public int HeaderValueLength { get; set; }
+
+        [GlobalSetup]
+        public void Setup()
+        {
+            var rand = new Random(42);
```

Comment:
Nothing random in performance tests please. We're trying to measure differences between implementations so use a couple of fixed headers from real requests.